backend/src/main/java/capstone/backend/mapper/ProductMapper.java

```diff
@@ -41,10 +44,25 @@ public static ProductDTO mapProductWithDetails(Product product){
                 .minAmount(product.getMinAmount())
                 .maxAmount(product.getMaxAmount())
                 .unitSize(product.getUnitSize())
-//                .suppliers(product.getSuppliers()
-//                        .stream()
-//                        .map(SupplierMapper::mapSupplier)
-//                        .toList())
+                .suppliers(mapProductList(product))
                 .build();
     }
+
+    private static List<Supplier> mapProductList(ProductDTO product) {
```

Comment:
Personally I would name the ProductDTO productDTO to differentiate from Product product, but I guess that's a matter of taste